frontend/app/views/support/Event.scala

```diff
@@ -0,0 +1,33 @@
+package views.support
+
+import model.Eventbrite.EBEvent
+
+object Event {
+
+  val ImportantClassName = "event-trait--important"
+  val ModerateClassName = "event-trait--moderate"
+  val NoteWorthyClassName = "event-trait--noteworthy"
+
+  case class Attribute(text: String, value: Boolean, className: String = "")
+
+  case class EventAttributes(
+    free: Attribute,
+    noDiscount: Attribute,
+    notSoldThroughEventbrite: Attribute,
+    soldOut: Attribute
+  )
+
+  def eventDetail(event: EBEvent) = {
+      val free = !event.isNoTicketEvent && event.generalReleaseTicket.exists(_.free)
```

Comment:
I feel like a fair bit of this logic (`free` & `live`) could live against the EBEvent class itself - the other two fields, `notSoldThroughEventbrite` and `soldOut` already do.